vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/server/business/ExternalPlatformBusiness.java

```diff
@@ -118,8 +121,12 @@ public ParseResult parseParameter(
                         externalPlatform, parameterName, parameterValue
                 );
                 return new ParseResult(true, shanoirUri);
+            case SRM:
+			    String srmUri = srmStorageBusiness.generateUri(externalPlatform,
```

Comment:
Beware of indents : need 4 spaces, no tabs.

Reply:
intends updated